### DISCUSSION TITLE
hotspot not showing up

### DISCUSSION THREAD

**Discussion by SCUZNUTS** [wayne-love/sn_esp32#9] [Q&A]
Should the wifi hotspot show up when powered over USB?
I have no buck converter yet so I'm just testing.

After flashing with platformio in VS Code, everything is successful.
I notice a solid red light and a flashing blue light, About 4 pulses.
No hotspot.

Should I be expecting hotspot etc to show up at this point?

**Comment by wayne-love** [comment]
The hotspot only starts once the enable button is pressed.  The blue flashing light should stop flashing IIRC when the device is in hotspot mode.
